pkg/cloudprovider/aws/instancetypes.go

```diff
@@ -85,6 +86,10 @@ func (p *InstanceTypeProvider) Get(ctx context.Context, provider *v1alpha1.AWS)
 			instanceType.AvailableOfferings = offerings
 			result = append(result, instanceType)
 		}
+		if !injection.GetOptions(ctx).AWSENILimitedPodDensity {
+			instanceType.MaxPods = new(int32)
```

Comment:
instanceType.MaxPods = ptr.Int32(110)

Reply:
I knew there had to be a better way to do that. Thanks 👍